src/LineChart/components/LineChart.ts

```diff
@@ -274,4 +323,27 @@ export class LineChart extends Component<LineChartProps, LineChartState> {
 
         return traces;
     }
+
+    public static getMarkerSizeReference(props: LineChartProps, series: LineSeriesProps, markerSize: number[], dimensions?: { width: number; height: number }): number {
+        if (series.autoBubbleSize) {
+            const width = dimensions ? dimensions.width : 0;
+            const height = dimensions ? dimensions.height : 0;
+            let sizeRef = 0;
+            const averageSize = (width + height) / 2;
+            const percentageSize = averageSize / 10;
+
+            if (markerSize.length > 0) {
+                sizeRef = Math.max(...markerSize) / percentageSize;
+            }
+
+            return Math.round(sizeRef * 1000) / 1000;
+        } else if (series.markerSizeReference > 0) {
```

Comment:
Integer modeler properties van not be < 0, so remove check
Or is this code called for non bubble charts too?

Reply:
But it can be 0. The check is to avoid an Infinity result that arises from dividing by 0.